docs/quickstarts/baremetal.md

```diff
@@ -0,0 +1,283 @@
+# Lokomotive Bare Metal Installation Guide
+
+## Contents
+
+* [Introduction](#introduction)
+* [Requirements](#requirements)
+* [Step 1: Install Lokoctl](#step-1-install-lokoctl)
```

Comment:
Maybe we can have "installation" paragraph here and make all steps as sub sections?

Reply:
I think the current contents section and heading look more focused.

Also , all the steps cannot be generalized into `Installation`.